core/server/api/canary/utils/validators/input/schemas/members-upload.json

```diff
@@ -0,0 +1,49 @@
+
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "members.upload",
+    "title": "members.upload",
+    "description": "Schema for members.upload",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "additionalProperties": true,
+        "required": ["email"],
+        "properties": {
+            "name": {
+                "type": "string",
+                "maxLength": 191,
+                "pattern": "^([^,]|$)"
+            },
+            "email": {
+                "type": "string",
+                "minLength": 1,
+                "maxLength": 191,
+                "format": "email"
```

Comment:
The "email" format here catches most basic cases, but is not as restrictive as our validator used in the model layer (the isEmail validation). I went with this one for now and we probably will need to go through each record again and call strict validator just in case? Haven't measure overhead on that, but expect it to be pretty big when working with >50K sets.

Reply:
Did a measurement to have some number in place. With 50k records it took ~160ms. Probably worth adding given this piece of data is important :thinking: